contracts/libraries/Logic.sol

```diff
@@ -305,4 +306,16 @@ library Logic {
         uint256 pricePerLP = lpPriceHolder.lp_price();
         return pricePerLP.mulDiv(FixedPoint128.Q128, 10**30); // 10**6 / (10**18*10**18)
     }
+
+    function migrate() external {
+        ICurveGauge oldGauge = ICurveGauge(0x97E2768e8E73511cA874545DC5Ff8067eB19B787);
+        ICurveGauge newGauge = ICurveGauge(0x555766f3da968ecBefa690Ffd49A2Ac02f47aa5f);
+
+        uint256 bal = oldGauge.balanceOf(address(this));
+
+        IERC20(0x8e0B8c8BB9db49a46697F3a5Bb8A308e744821D2).approve(address(oldGauge), 0);
```

Comment:
Here I think we should mention which token this is, probably in a comment or just create an intermediate variable variable.
```suggestion
        IERC20 crv3 = IERC20(0x8e0B8c8BB9db49a46697F3a5Bb8A308e744821D2);
        crv3.approve(address(oldGauge), 0);
```